### ISSUE TITLE
Default values for primitives features

### ISSUE THREAD

**Issue by AlArgente** [webanno/webanno#1687]
Hi, I'm trying to use constraints for primitives features to make them to have a default value while annotating. I want to use this because I want to make a double document level annotation, but I don't want to use zero-width annotation because it seems to be hard to annotate for some annotators and it requires a lot of time. 

I know that at Inception I can do this type of annotation but it won't count for calculating the metrics, so I didn't swap to Inception.

My objetive is to reduce the time the annotation takes, so I think this could be helpful. Right know I'm doubting if I must use a string feature and a slot feature with default values that Webanno provide, or using two slot features, one for each document level annotation.

Sorry if I explained it wrong, my English is not very good.

I think Webanno is a really good annotation system, and I think that this kind of feature could help to improve the performance of it. 

Best,
Alberto.

**Comment by reckart** [comment]
Would it be sufficient for you to add an option to set a default value for a feature in the feature configuration of the layers tab in the project settings?

**Comment by AlArgente** [comment]
But I can only do that if the feature is a link to a layer. I did that in the end and seems to work fine. I just thought it could be good to have default values for string's features, so when you select the spans then you just click and it get auto-annotated.

Thanks for your help.

**Comment by reckart** [comment]
So let's say you have an annotation with some string feature. You want that you can configure a default value for that string feature such that when you create en annotation of that type, the string feature starts with the default value, right?

**Comment by AlArgente** [comment]
Yes. But I want this type of annotation for a document level annotation, so I will use it just 1 time for each document. I know I can have a default value if I multiples annotations from a string feature, and its really useful but not in this case.

As I thought it is like the default value for slots features, but without having to click on the word again.

**Comment by jianlins** [comment]
Has this feature been implemented?

**Comment by reckart** [comment]
Not yet. WebAnno isn't updated anymore anyway.

But it seems like a good time to start adding this to its successor INCEpTION.

https://github.com/inception-project/inception/issues/5429

**Comment by reckart** [comment]
@jianlins if you need this for non-string features, please comment on the INCEpTION issue linked above.

**Comment by jianlins** [comment]
Oh, I posted on wrong repo.
